### DISCUSSION TITLE
mmWave Radar Sensor Guide | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1173] [Q&A]
# mmWave Radar Sensor Guide | Seeed Studio Wiki

mmWave Radar Sensor Guide

https://wiki.seeedstudio.com/select_lorawan_network/

**Comment by eerke2021** [comment]
Does the M2 linked above emit MQTT messages? I’m pretty sure chirpstack can produce MQTT messages. It’ll it does what is the Topic? Thanks!
